app/views/home/contributors.md

```diff
@@ -0,0 +1,69 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that might be looking for ways to use those skills on an open source project.
```

Comment:
> But for now, I think scoping it to people with existing software skills is the right way to go

👍 

> This matches up with our objectives, as currently defined.

To others following along: @bkeepers, @kytrinyx, @nayafia and I work on the open source team at @GitHub and our referenced 2017 objective is to increase the number of people on GitHub contributing to public, licensed repositories in a given week.

> If so, should we create a guide for that topic? Then we could link to that guide if we want to start working towards encouraging other skill sets.

👍 Again, for others following along this refers to https://opensource.guide which anyone can contribute to at https://github.com/github/opensource.guide.